### PR TITLE
[release] Linux and psql

### PR DESCRIPTION
**Pull request by haotianzhu**

### CODE REVIEW

File: script/host_info.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+cpu_mhz=$(echo "$lscpu_out"  | egrep "CPU MHz:" | awk '{print $3}' | xargs)
+
+l2_cache=$(echo "$lscpu_out"  | egrep "^L2 cache:" | awk '{print $3}' | xargs)
+
+
+
+
+total_mem=$(echo "$meminfo" | egrep "^MemTotal:" | awk '{print $2}' | xargs)
+
+timestamp=$(vmstat -t | grep -o -e '[0-9]\{4\}\-.*' | xargs)
```
Comment (jarviscanada) [inline script/host_info.sh]: remove all unnecessary new lines for better readability purposes

File: script/host_info.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+#usage
+memory_free=$(echo "$meminfo" | egrep "^MemFree:" | awk '{print $2}' | xargs)
+cpu_idle=$(vmstat -t | grep  -Po '\d+(?=\s+\d+\s+\d+\s+\d+\-.*)')
+cpu_kernel=$(vmstat -t | grep  -Po '\d+(?=\s+\d+\s+\d+\s+\d+\s+\d+\-.*)')
+disk_io=$(vmstat -d | grep  -Po '\d+(?=\s+\d+$)')
+disk_available=$(df -BM / | grep  -Po '\d+(?=M\s+\d+%)' )
+
+# testing
+#echo "$hostname|$cpu_number|$cpu_architecture|$cpu_model|$cpu_mhz|$l2_cache|$cpu_idle|$cpu_kernel|$disk_io|$disk_available"
```
Comment (jarviscanada) [inline script/host_info.sh]: you need to insert data to PSQL database

**Review by jarviscanada — APPROVED** [review]
fix the comment and merge

File: script/host_info.sh (hunk 2 above)
Comment (haotianzhu) [reply script/host_info.sh]: Yeah, the monitor agent was not finished yet last week.  I just finished #7  today.

File: script/host_info.sh (hunk 1 above)
Comment (haotianzhu) [reply script/host_info.sh]: I fixed them

**Pull request merged**